README.md

```diff
@@ -177,14 +177,22 @@ $EDITOR config/secrets.json
 # Otherwise stub it:
 echo "{}" > config/crowdtangle_list.json
 
-# If detectHateSpeech is set to true, then run the python 2 hate-speech-api.
+# Follow these steps for setting up hate speech API for Burmese
+python --version # check if you have python 2 installed.
 cd hate-speech-api
-pip install -r requirements.txt
-# ONLY IF the above command doesn't work, use instead:
-# python2 -m pip install -r requirements.txt
-
-# Start the hate-speech-api server.
-python2 hate_speech_clf_api.py
+npm install forever -g # install `forever` npm module.
+pip install virtualenv # install virtual environment.
+virtualenv venv # create virtual environment.
+source venv/bin/activate # activate virtual environment
+pip install -r requirements.txt # install dependencies 
+# User input: Set `detectHateSpeech: true`.
+$EDITOR config/secrets.json
+# User input: Set the script to run on startup.
+crontab -e
+# Paste the following line in crontab:
+@reboot forever start -c python -e error.log hate_speech_clf_api.py &
+# Reboot the machine and make sure the Hate Speech API is available on port 5000:
+curl localhost:5000
```

Comment:
Will this return a meaningful result? Please adjust as needed so an admin can confirm the API is indeed running

Reply:
done. Thanks for the instructions.